src/pages/SignUpPage/styles.js

```diff
@@ -1,23 +1,21 @@
-const styles = () => ({
+const styles = theme => ({
   root: {
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
-  },
-  signUpCard: {
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
-    width: '536px',
-    marginTop: '37px',
-    marginBottom: '37px',
-    paddingBottom: '20px',
+    [theme.breakpoints.up('sm')]: {
```

Comment:
What is this? Just curious cuz I really don't know LOL

Reply:
https://material-ui.com/customization/breakpoints/

Reply:
just a nice wrapper around media queries that mui gives us